Lib/pydoc.py

```diff
@@ -631,15 +632,19 @@ class HTMLDoc(Doc):
 
     def page(self, title, contents):
         """Format an HTML page."""
+        css_data = importlib.resources.files('pydoc_data').joinpath('_pydoc.css').read_text()
```

Comment:
Can this fail on any envs?

Reply:
Ideally rhis should also be documented (i.e. you should say which file we are looking for). Maybe we can have a list of allowed files?